src/Html/Options/HasColumns.php

```diff
@@ -43,14 +52,14 @@ public function columns(array $columns)
                 if (is_array($value)) {
                     $attributes = array_merge(
                         [
-                            'name' => $value['name'] ?? $value['data'] ?? $key,
+                            'name' => $value['name'] ?? $this->camelRelation($value['data'] ?? $key),
```

Comment:
I think this is not needed on this part since not all name are on relation. This might fail if the column is indeed have underscores on it? I think it's better to do a camelCase on the part where eagerLoading is being checked.

Reply:
Or this might also fail if the query uses join statements? Anyways, will also try to check on actual projects when I got chance. Thanks!

Reply:
Well, the camelRelation function only applies the camel case to the relation part of the name. For example,  camelRelation('last_name') ->  'last_name',  but camelRelation('job_contacts.last_name') -> 'jobContacts.last_name'

And of course if the 'name' is actually specified in the columns array to begin with, it will not be altered.

I will see if I can write some  test classes for this when I get a chance, as I think it would really help to show things like the query join situations you mention.

Reply:
Yes, a test would be great. For join queries, it is advised to include `table.column` when setting the name which is why this might become a potential breaking change.